contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -31,7 +31,6 @@
 //  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 //////////////////////////////////////////////////////////////////////////
-
```

Comment:
Can we re-instate this line, we try to keep any whitespace changes in their own commits if they're required.